### PR TITLE
Editing applications

### PR DESCRIPTION
**Pull request by Preeticp**
- This PR adds the editing application workflow.
- It has been verified by SMEs and the QE
- It is relevant for the 4.4 version.

### CODE REVIEW

File: modules/odc-editing-application-configuration-using-developer-perspective.adoc [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+// applications/application_life_cycle_management/odc-editing-applications.adoc
+
+[id="odc-editing-application-configuration-using-developer-perspective_{context}"]
+= Editing the application configuration using the Developer perspective
+
+You can use the the *Topology* view in the *Developer* perspective to edit the configuration of your application.
+
+[NOTE]
+====
+Currently, only configurations of applications created by using the *From Git*, *Container Image*, *From Catalog*, or *From Dockerfile* options in the *Add* flow of the *Developer* perspective can be edited. Configurations of applications created by using the CLI or the *YAML* option from the *Add* flow can not be edited.
```
Comment (ahardin-rh) [inline modules/odc-editing-application-configuration-using-developer-perspective.adoc]: s/flow/workflow
s/can not/cannot

File: modules/odc-editing-application-configuration-using-developer-perspective.adoc [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+====
+
+.Prerequisite:
```
Comment (ahardin-rh) [inline modules/odc-editing-application-configuration-using-developer-perspective.adoc]: You can remove the : after Prerequisite

**Comment by ahardin-rh** [comment]
@Preeticp Just a few minor suggestions from me. Looks good!

**Comment by Preeticp** [comment]
@ahardin-rh fixed the comments and squashed the commits. Please merge, if you feel it is alright.

**Pull request merged**